daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2471,6 +2472,7 @@ func (devices *DeviceSet) Status() *Status {
 		}
 	}
 
+	status.MinFreeSpace = fmt.Sprintf("%v%%", devices.minFreeSpacePercent)
```

Comment:
Open to suggestions; my train of thought was that "percentage" is a convenient way to define the amount of free space, but since all other values shown in `docker info` are absolute values, possibly the calculated value (`... MB`) is more easy to use in that context.

So, a bit on the fence on that one; perhaps someone else has suggestions